### DISCUSSION TITLE
Limit in number of creation of projects and fetch

### DISCUSSION THREAD

**Discussion by aejazuddinam** [DependencyTrack/dependency-track#1193] [Q&A]
Hi,

I have a query to ask,
1. How many maximum number of projects can be created in Dependency Track, is there any limit ?
2. How many maximum number of projects can be fetched from REST API call ? and how to fetch the full maximum limit.

**Comment by stevespringett** [comment]
> How many maximum number of projects can be created in Dependency Track, is there any limit ?

There is no limit

> How many maximum number of projects can be fetched from REST API call?

By default, the API will return a maximum of 100 records . This can be changed.

> how to fetch the full maximum limit

Refer to #209 for examples.
